samples/SocialSample/Startup.cs

```diff
@@ -228,14 +191,42 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
 
                         context.RunClaimActions(user);
                     }
+                };
+            });
+
+            services.AddAuthentication(options => options.DefaultAuthenticationScheme = CookieAuthenticationDefaults.AuthenticationScheme);
+        }
+
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
+        {
+            loggerfactory.AddConsole(LogLevel.Information);
+
+            // Simple error page to avoid a repo dependency.
+            app.Use(async (context, next) =>
```

Comment:
Can we just have Security depend on Diagnostics so we can stop doing this stuff? I don't think it creates any cycles, it just bumps security up the chart one level.